psychopy/sound/backend_ptb.py

```diff
@@ -332,13 +332,19 @@ def __init__(self, value="C", secs=0.5, octave=4, stereo=-1,
         self.setSound(value, secs=self.secs, octave=self.octave,
                       hamming=self.hamming)
         self._isPlaying = False  # set `True` after `play()` is called
+        self._isFinished = False
         self.status = NOT_STARTED
 
     @property
     def isPlaying(self):
         """`True` if the audio playback is ongoing."""
         return self._isPlaying
 
+    @property
+    def isFinsihed(self):
```

Comment:
Typo in the name of the definition (isFinsihed ). Makes the code break

Reply:
Thanks @asoskic 🤦 😆

Reply:
https://github.com/psychopy/psychopy/pull/5627